src/DataTable/ExpanderRow.js

```diff
@@ -9,22 +9,22 @@ const renderChildren = (children, data) => {
 };
 
 const ExpanderRowStyle = styled.div`
+  flex: 0 0 100%;
   width: 100%;
   box-sizing: border-box;
   color: ${props => props.theme.expander.fontColor};
   background-color: ${props => props.theme.expander.backgroundColor};
 `;
 
-const ExpanderRow = ({
```

Comment:
We should preserve the original formatting to keep not only to keep the diff clean but to be consistent with other components